cyences_app_for_splunk/default/savedsearches.conf

```diff
@@ -2043,6 +2144,7 @@ request.ui_dispatch_app = cyences_app_for_splunk
 request.ui_dispatch_view = search
 search = `cs_o365` sourcetype="ms:o365:reporting:messagetrace" _index_earliest=-25h@m _index_latest=-1h@m Status=FilteredAsSpam \
 | stats count, values(RecipientAddress) as RecipientAddress by SenderAddress, Subject \
+| eval cyences_severity = "info" \
```

Comment:
This alert is specific to Office 365 but the alert name suggests a generic naming convention.
Let's discuss this.